From 57a6797c1513f1cbf29047b1e95c9711216d8ad5 Mon Sep 17 00:00:00 2001
From: Utsav Shankar <utsaveshan0206@gmail.com>
Date: Mon, 13 Jul 2026 23:05:44 +0800
Subject: [PATCH 05210/15333] Documentation: usb: remove dead link to external
 USB programming guide

The "Programming Guide for Linux USB Device Drivers" was last hosted
at lmu.web.psi.ch and is no longer reachable. The original source
(TUM, cs.tum.edu) is also defunct. Remove the dead reference rather
than link to a third-party archive of a 20+ year old, outdated guide.

Signed-off-by: Utsav Shankar <utsaveshan0206@gmail.com>
Signed-off-by: Jonathan Corbet <corbet@lwn.net>
Message-ID: <20260713150544.20962-1-utsaveshan0206@gmail.com>
---
 Documentation/driver-api/usb/writing_usb_driver.rst | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/Documentation/driver-api/usb/writing_usb_driver.rst b/Documentation/driver-api/usb/writing_usb_driver.rst
index 95c4f5d14052..6f024e1647cd 100644
--- a/Documentation/driver-api/usb/writing_usb_driver.rst
+++ b/Documentation/driver-api/usb/writing_usb_driver.rst
@@ -322,7 +322,4 @@ http://linux-hotplug.sourceforge.net/
 linux-usb Mailing List Archives:
 https://lore.kernel.org/linux-usb/
 
-Programming Guide for Linux USB Device Drivers:
-https://lmu.web.psi.ch/docu/manuals/software_manuals/linux_sl/usb_linux_programming_guide.pdf
-
 USB Home Page: https://www.usb.org
-- 
2.55.0

